packages/expo-image-picker/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### 💡 Others
 
+## 14.3.1 — 2023-06-29
+- Fix manifest merger build fail on Android. ([#23191](https://github.com/expo/expo/pull/23191) by [@alexandrius](https://github.com/alexandrius))
```

Comment:
Can you please remove the version and date from here and move the entry a few lines up to be under bug fixes section?

Reply:
Sorry for these. Will be more careful next time.